src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -273,7 +277,7 @@ public function configureOptions(OptionsResolver $resolver)
      *
      * @return EntityLoaderInterface
      */
-    abstract public function getLoader(ObjectManager $manager, $queryBuilder, $class);
+    abstract public function getLoader(LegacyObjectManager $manager, $queryBuilder, $class);
```

Comment:
changing the type would be a BC break

Reply:
Would it really? It's a class alias, so they are the same thing, right?

Reply:
Not when v1.2 is checked out